src/elements/Image/Image.js

```diff
@@ -1,28 +1,38 @@
-import React, { Component, PropTypes } from 'react'
-import classNames from 'classnames'
+import React, { PropTypes } from 'react'
+import cx from 'classnames'
 import META from '../../utils/Meta'
 
-export default class Image extends Component {
-  static propTypes = {
-    alt: PropTypes.string,
-    className: PropTypes.string,
-    src: PropTypes.string,
-  }
-
-  static _meta = {
-    library: META.library.semanticUI,
-    name: 'Image',
-    type: META.type.element,
-  }
-
-  render() {
-    const classes = classNames(
-      'sd-image',
-      'ui',
-      this.props.className,
-      'image'
-    )
-
-    return <img {...this.props} className={classes} />
-  }
+function Image(props) {
+  const {
+    className,
+  } = props
+
+  const classes = cx(
+    'sd-image', 'ui',
```

Comment:
Heads up, I've finished removing all `sd-*` classes, see #301.  Rebase to master to get the update.  The test for the `sd-*` class has been inverted to fail if any `sd-` class exists.

Let's remove all `sd-*` classes from all PRs.  👍